packages/manager/apps/pci-kubernetes/src/components/Autoscaling.component.tsx

```diff
@@ -76,10 +85,43 @@ export function Autoscaling({
       min: initialScaling ? initialScaling.min : 0,
       max: initialScaling ? initialScaling.max : NODE_RANGE.MAX,
     });
-  }, [isAutoscale]);
+  }, []);
 
   return (
     <>
+      <QuantitySelector
+        className="mt-4"
+        label={t('kubernetes_node_pool_autoscaling_desired_nodes_size')}
+        value={quantity.desired}
+        onValueChange={(desired) => {
+          setQuantity((_quantity) => ({
```

Comment:
Prefer using something else than `_` because it is commonly used for unused parameters

You can use oldQuantity or simply q